app/client/components/billing/invoicesTable.tsx

```diff
@@ -31,6 +32,7 @@ interface InvoicesTableProps {
 }
 
 export const InvoicesTable = (props: InvoicesTableProps) => {
+  let trackingPaginationInteractionCount = 1;
```

Comment:
do we definitely need a mutable variable? should it not be a piece of state rather than a `let`